### ISSUE TITLE
Consider allowing anything as an argument that implements `Display`

### ISSUE THREAD

**Issue by casey** [soenkehahn/stir#29]
This would be a large change to the API, so it's worth considering up-front.

This would let users pass things like integers without having to call to_string up front. Additionally, users could pass their own types if they implemented `Display` for them.

**Comment by soenkehahn** [comment]
Yeah, that's a really good idea. I think it'd be worth creating a PR that implements this to see whether this clashes with anything that's implemented or tested so far.
